primitives/src/assets.rs

```
@@ -474,6 +474,6 @@ where
 			)
 			.map(|_| ())
 		}
-		.map_err(|e| FungibleLedgerError::InvalidTransfer(e))
```

Why not propagate the dispatch error ? It will have useful info.

---

The code has the same semantics :p
Nothing has been changed.

---

See previous conversation between @Garandor  and @Dengjianping